src/ast/mod.rs

```diff
@@ -119,16 +203,18 @@ where
     PrintOverallStatistics,
     Check(Span, Vec<GenericFact<Head, Leaf>>),
     CheckProof,
-    PrintTable(Symbol, usize),
-    PrintSize(Option<Symbol>),
+    PrintTable(Span, Symbol, usize),
+    PrintSize(Span, Option<Symbol>),
     Output {
+        span: Span,
         file: String,
         exprs: Vec<GenericExpr<Head, Leaf>>,
     },
     Push(usize),
```

Comment:
How did you decide what gets a span? Looks like Push doesn't have one. Perhaps because there are no error messages with it?

Reply:
Yeah there's no error message associated with `push` right now. For now I just added spans to commands that may report a `TypeError`.